### PR TITLE
Case sensitive

### PR DESCRIPTION
**Pull request by aaronmatson1**
## Type of change
Please delete options that are not relevant.
- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
## Change Status
- [X] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback
# Checklist
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/components/common/searchbar/index.js [hunk 1]
```diff
@@ -8,6 +8,8 @@ function SearchBar({ searchVisible, setData }) {
   const { Search } = Input;
   const { Option } = Select;
 
+  console.log('hello from search bar');
```
Comment (lgv-0) [inline src/components/common/searchbar/index.js]: Let's drop this 👍

**Review by lgv-0 — CHANGES_REQUESTED** [review]

**Review by lgv-0 — APPROVED** [review]

File: src/components/common/searchbar/index.js [hunk 2]
```diff
@@ -53,7 +53,7 @@ function SearchBar({ searchVisible, setData }) {
           placeholder="Search through your inventory"
           className="searchBar"
           onSearch={onSearch}
-          onChange={onChange}
+          onChange={e => onSearch(e.target.value)}
```
Comment (edelveiss) [inline src/components/common/searchbar/index.js:53]: We have already onChange function:
 function onChange(e) {
    setData(e.target.value);
  }
So, we do not need this change. Also we have already OnSearch function for searching after Enter key:
function onSearch(values) {
    setData(values);
  }

**Review by edelveiss — COMMENTED** [review]
Now we have already a version that works and is case insensitive, so a user can work with upper- and lowercase letters.

File: src/components/common/searchbar/index.js (hunk 2 above)
Comment (aaronmatson1) [reply src/components/common/searchbar/index.js:53]: Alright. Just wanting to try and filter it down as the user is typing, But it is probably better after they enter the full term.

**Review by Dustnclay — APPROVED** [review]
most likely the best pull request I've ever seen

**Pull request merged**